### DISCUSSION TITLE
Tracing support for slim framework

### DISCUSSION THREAD

**Discussion by leoleva** [getsentry/sentry-php#1748] [Q&A] [answered]
Hello, I'm searching for a sentry tracing integration for the slim framework. Does sentry support such integration? If not I would be interested to create one but as I'm quite new to all of this sentry tracing things would be nice to know if there are any articles about best practices on how to achieve this? Thanks

**Comment by stayallive** [comment, marked as answer]
I am not aware of any integrations for that framework. 

The best resources are the docs: https://docs.sentry.io/platforms/php/performance/

And the other two framework integrations for some actual code:
- https://github.com/getsentry/sentry-laravel
- https://github.com/getsentry/sentry-symfony
